src/Sylius/Bundle/CoreBundle/Form/Extension/LocaleTypeExtension.php

```diff
@@ -1,77 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sylius package.
- *
- * (c) Sylius Sp. z o.o.
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-declare(strict_types=1);
-
-namespace Sylius\Bundle\CoreBundle\Form\Extension;
-
-use Sylius\Bundle\LocaleBundle\Form\Type\LocaleType;
-use Sylius\Component\Locale\Model\LocaleInterface;
-use Sylius\Component\Resource\Repository\RepositoryInterface;
-use Symfony\Component\Form\AbstractTypeExtension;
-use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\Form\FormEvent;
-use Symfony\Component\Form\FormEvents;
-use Symfony\Component\Intl\Locales;
-
-final class LocaleTypeExtension extends AbstractTypeExtension
```

Comment:
This extension should be deprecated in 1.14 if we remove it

Reply:
#16539